### ISSUE TITLE
NTR: RPR positive in blood (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8760]
New term request RPR positive in blood (nadea3_181018023046): Rapid plasma reagin test  UMLS: 0240917

**Comment by pnrobinson** [comment]
see https://www.ncbi.nlm.nih.gov/books/NBK557732/

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000797
- label: Positive bloodstream rapid plasma reagin test
- parents: [Bloodstream infectious agent (HP:0031863)](https://hpo.jax.org/app/browse/term/HP:0031863)
- pmid: 
- def: Positive result of the the rapid plasma reagin test (RPR test. This is a rapid diagnostic test that looks for non-specific antibodies in the blood of the patient that may indicate an infection by syphilis or related non-venereal treponematoses.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8760
- synonyms:
